src/app/components/client/ScanResultCard.tsx

```diff
@@ -0,0 +1,287 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import React, { ReactNode, useId } from "react";
+import { OnerepScanResultRow } from "knex/types/tables";
+import styles from "./ExposureCard.module.scss";
+import { StatusPill, StatusPillType } from "../server/StatusPill";
+import {
+  ChevronDown,
+  EmailIcon,
+  LocationPinIcon,
+  MultipleUsersIcon,
+  PhoneIcon,
+} from "../server/Icons";
+import { useL10n } from "../../hooks/l10n";
+import { ExposureCardDataClassLayout } from "./ExposureCardDataClass";
+import { DataBrokerImage } from "./DataBrokerImage";
+import { CONST_URL_SUMO_MONITOR_REMOVAL } from "../../../constants";
+import { TelemetryLink } from "./TelemetryLink";
+
+export type ScanResultCardProps = {
+  scanResult: OnerepScanResultRow;
+  locale: string;
+  isPremiumBrokerRemovalEnabled: boolean;
+  resolutionCta: ReactNode;
+  isPremiumUser: boolean;
+  isExpanded: boolean;
+  isOnManualRemovePage?: boolean;
```

Comment:
If components or functions have exceptions for a single caller, that's often an indication that the abstraction might not be a great fit - they can make things harder to follow, as half of the code you're looking at might be irrelevant to the particular situation you're investigating.

A way to avoid that is by lifting the logic up to the caller, i.e. have it pass in the string to render directly. That way, there's no need for an `if` anywhere.

(This isn't a blocker, but it's a good exercise to get a better feel for software architecture :slightly_smiling_face: )